lexicons/com/atproto/sync/getRepoStatus.json

```diff
@@ -27,7 +27,14 @@
             "status": {
               "type": "string",
               "description": "If active=false, this optional field indicates a possible reason for why the account is not active. If active=false and no status is supplied, then the host makes no claim for why the repository is no longer being hosted.",
-              "knownValues": ["takendown", "suspended", "deactivated"]
+              "knownValues": [
+                "takendown",
+                "suspended",
+                "deleted",
+                "deactivated",
+                "broken",
```

Comment:
flagging for discussion: didn't seem like folks don't love `broken` (though I don't mind it). earlier term we were using was `dirty`.

No great words have come up yet... `fixme`, `out-of-sync`, `unsynchronized`, `disjoint`?

Reply:
going with `desynchronized`